www/squid/src/opnsense/service/conf/actions.d/actions_proxy.conf

```diff
@@ -80,3 +81,9 @@ command:/usr/local/opnsense/scripts/proxy/download_error_pages.py
 parameters:
 type:script_output
 message:download error pages
+
+[export_keys]
+command:/usr/local/bin/zip -FSjr /tmp/squid-tlskeys.zip /var/log/squid/tlskey.log*
+parameters:
+type:script_output
```

Comment:
Likely better to switch to `stream_output` and let the zip command output to stdout. I don't mind a single file large blob download, but this will run out of memory at some point. 

For inspiration see https://github.com/opnsense/core/commit/cee33f5393154bb459150fd4639a6a8cd77456ee

or even simpler, implement the `configdStream()` to call `configd` and pass the results to the client
 https://github.com/opnsense/core/blob/e2e365df4bfa049bd10e124c8093dfb314c538d4/src/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php#L172-L178